examples/mixture_models/dirichlet_mixture_of_multinomials.ipynb

```diff
@@ -0,0 +1,1819 @@
+{
```

Comment:
<p>Besides the Generalized Dirichlet you can also mention the Logistic Multivariate Normal distribution: <a href="https://en.wikipedia.org/wiki/Logit-normal_distribution#Multivariate_generalization" target="_blank">https://en.wikipedia.org/wiki/Logit-normal_distribution#Multivariate_generalization</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>